acid/features/auth/controller.py

```diff
@@ -6,11 +6,11 @@
 
 from werkzeug.utils import redirect
 
-from dashboard.auth import service
-from dashboard.controller import error_handlers
-from dashboard.auth.exceptions import AuthenticationFailed
+from acid.features.auth import service
+from acid.controller import error_handlers
+from acid.features.auth.exceptions import AuthenticationFailed
```

Comment:
Instead of ``acid.features.auth.exceptions`` use relative import like ``.exceptions`` for modules which are in the same catalog.

Please change it in all features, long imports path are not good.

Reply:
ready